src/LayerPlan.cpp

```diff
@@ -1514,12 +1532,28 @@ TimeMaterialEstimates ExtruderPlan::computeNaiveTimeEstimates(Point starting_pos
     TimeMaterialEstimates ret;
     Point p0 = starting_position;
 
+    double min_path_speed = fan_speed_layer_time_settings.cool_min_speed;
+
+    //TODO: This slowest_path_speed is causing warnings for empty layers and looks a bit uckly to me anyway.
```

Comment:
Any suggestions for how to fix this todo?

if it was python i would have written: slowest_path_speed = min([path.config->getSpeed().value * path.speed_factor for path in paths if not path.isTravelPath()])

Reply:
I think you forgot to check for `isTravelPath()` in the loop.

I've fixed that, as well as the recalculation. The value is now just a class variable.

I've also used the 'accumulate' function to be a bit closer to what python is doing. There is another way to get even closer (using 'ranges'), but I found that in this case it was more hassle than it's worth. It doesn't matter that much in a simple case like this anyway.